__tests__/components/WhatWeDoItem/WhatWeDoItem.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
WhatWeDoItem.test.tsx inside of the WhatWeDoItem folder is not a proper way of this file location.

Reply:
I just follow the way that previous commits did, like Button.test.tsx in Button folder. Should I just put test files under __test__/components ?

Reply:
> 
Blindly copying and pasting is not always good and right. When WhatWeDoItem.test.tsx inside of the same name folder, it duplicated and caused confusion for other devs. Consider using "WhatWeDoItem/index.tsx" or just without same name parent folder.